apps/funke/app/onboarding/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { Redirect } from 'expo-router'
+import { Text, View } from 'react-native'
+
+import { initializeAppAgent, useSecureUnlock } from '@/agent'
```

Comment:
sometimes its @package and sometimes its @/ is there a reason for this?

Reply:
`@/` is inside the app. But it's a bit confusing. We can do `@app`